frontend/src/utils/csvToPropertyModel.ts

```diff
@@ -124,39 +128,51 @@ export const parseCSVString = async (csvContent: string): Promise<IPropertyModel
 
     const getValueOrDefault = (incomingValue: string | undefined, defaultValue: string) =>
       !incomingValue || incomingValue === '' ? `${defaultValue}` : incomingValue;
+
+    console.log('test parsedCSV', parsedCSV);
+
     // Transform raw objects into model that API expects
     const transformedData: IPropertyModel[] = parsedCSV.data.map(
-      (property: IExportedPropertyModel) => ({
-        parcelId: property.PID,
-        pid: property.PID,
-        pin: property.PIN ?? '',
-        status: getValueOrDefault(property.Status, 'Active'), // Assume active if not specified
-        fiscalYear: getFiscalYear(property),
-        agency: '', // Not used in API. Leave blank.
-        agencyCode: property.Ministry, // Names are misleading here.
-        subAgency: property.Agency ?? '',
-        propertyType: property.Type,
-        localId: property['Local ID'] ?? '',
-        name: property.Name,
-        description: property.Description ?? '',
-        classification: property.Classification,
-        civicAddress: property.Address ?? '',
-        city: getValueOrDefault(property.Location, '<blank>'), // Done to ensure something is passed to backend, empty string not enough
-        postal: property.Postal ?? '',
-        latitude: property.Latitude,
-        longitude: property.Longitude,
-        landArea: getValueOrDefault(property['Land Area'], '0'),
-        landLegalDescription: property['Legal Description'] ?? '',
-        buildingFloorCount: getValueOrDefault(property['Building Floor Count'], '1'),
-        buildingConstructionType: getValueOrDefault(property['Construction Type'], 'Unknown'),
-        buildingPredominateUse: getValueOrDefault(property['Predominate Use'], 'Unknown'),
-        buildingTenancy: property.Tenancy ?? '',
-        buildingRentableArea: getValueOrDefault(property['Rentable Area'], '0'),
-        assessed: getAssessedValue(property),
-        netBook: getValueOrDefault(property['Netbook Value'], '0'),
-        regionalDistrict: getValueOrDefault(property['Regional District'], ''),
-      }),
+      (property: IExportedPropertyModel) => {
+        const city = existingAdminAreas.some((adminArea) => adminArea.name === property.Location)
+          ? property.Location || ''
+          : getValueOrDefault(property['Regional District'], '');
```

Comment:
Use the value from the Location column of the spreadsheet if it exists in the administrative areas in PIMS, if not then use the value from the Regional District column.